src/main/java/duke/Duke.java

```diff
@@ -17,14 +19,23 @@ public static void main(String[] args) {
         System.out.println(separateLine);
 
         String line="";
+        List<String> list = new ArrayList<String>();
         Scanner in = new Scanner(System.in);
         while(!( line = in.nextLine()).equals("bye")) {
-                System.out.println(separateLine + "\n" + line + "\n" + separateLine);
+                if(!line.equals("list")){
+                    list.add(line);
+                    System.out.println(separateLine + "\n" + "Added: " + line + "\n" + separateLine);
+                }else{
+                    System.out.println(separateLine);
+                    for(int i = 1; i <= list.size(); i++)
+                    {
```

Comment:
Can consider sticking to one indentation style, it will be easier to read when dealing with a larger codebase or working long hours on the project.